src/main/java/edu/oakland/arttour/controller/ArtTourController.java

```diff
@@ -253,13 +253,13 @@ public void addToTour(
   }
 
   @CrossOrigin
-  @PostMapping("tour/{tourId}/artwork/{artworkId}/removal")
+  @PostMapping("tour/{tourId}/artwork/removal")
   public void deleteFromTour(
-      HttpServletRequest request, @PathVariable int tourId, @PathVariable String artworkId)
+      HttpServletRequest request, @PathVariable int tourId, @RequestBody List<String> artworkIds)
       throws SoffitAuthException {
     String email =
         authorizer.getClaimFromJWT(request, "email").asString(); // just to check if token is valid
-    dao.deleteFromTour(tourId, artworkId);
```

Comment:
I don't like that I'm putting it in here but it was throwing me an error when I created a service. Anyway this is pretty short and shouldn't be that big of a deal

Reply:
There's already a service class that you should be able to put it in but yeah its not a big deal

Reply:
that's probably why it was erroring lol